### ISSUE TITLE
[Feature Request] Packaging chrome extensions

### ISSUE THREAD

**Issue by xenorio** [sarweshparajuli/nativefier-appimage#5]
A really great feature would be an option to package chrome extensions. For example, pointing the script to CRX files or even to the chrome webstore URLs.

This would open up a bunch of new possibilities. I'm personally thinking about a YouTube Music client with integrated adblock and an extension that automatically confirms the "yes, i'm still listening" popup.

**Comment by kdwk** [comment]
Agreed. Please add support for Firefox extensions too if this gets implemented

**Comment by xenorio** [comment]
> Agreed. Please add support for Firefox extensions too if this gets implemented

As this is running chromium (which, as the name suggests, is basically chrome) under the hood, firefox extensions would not be possible.

**Comment by kdwk** [comment]
Ah, of course it is running Chromium

**Comment by axelalt** [comment]
An ad block integrated in appimage

**Comment by Kyanoxia** [comment]
As this is basically an extension of [Nativefier](https://github.com/nativefier/nativefier/), please check out [this issue](https://github.com/nativefier/nativefier/issues/177) on their page.  It points to a solution (I can't figure out how to use it, but others have) for AdBlocking and also discusses using Chrome/Chromium extensions in the apps.
